packages/tools/devtools/devtools-view/src/test/tsconfig.cjs.json

```diff
@@ -2,14 +2,11 @@
 	// This config must be used in a "type": "commonjs" environment. (Use fluid-tsc commonjs.)
```

Comment:
Looks like comment should be kept. If it is used without `fluid-tsc commonjs`, then it generates ESM.

Reply:
I didn't remove it. It's still there, though github is highlighting the line in red for some reason.